examples/tests/invalid/package.json

```diff
@@ -3,7 +3,8 @@
   "dependencies": {
     "cdktf": "0.20.3",
     "constructs": "^10.3",
-    "jsii-code-samples": "1.7.0"
+    "jsii-code-samples": "1.7.0",
+    "jsii-fixture": "workspace:^"
```

Comment:
Just making sure you meant to add this

Reply:
Yes, this relates to using the jsii fixture in the invalid tests.